### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ New Project_ Keyboard: Keyboard focus is not visible on “Maven” menu item.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3356]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.4.7; #Keyboard; #Narrator; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** 2.4.7

**Note:** Same issue is observed for the list items as well.

**Repro Steps:**
1) Open IntelliJ IDEA application.
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup.
3) Tab till 'Plugins' tab and press enter.
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in.
7) Sign in with any Azure subscription.
8) Tab till File menu and click new-->Project.
9) In the New Project dialog box, select Maven.
10)Verify keyboard focus on “Maven” menu item.

**Actual Result:**
Keyboard focus is not visible on “Maven” menu item.

**Expected Result:**
Keyboard focus should be  visible on “Maven” menu item.

**User Impact:**
Keyboard users find difficulty in understanding where the focus is if focus is not visible on few controls.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}
![Bug_3356_Keyboard issue](https://user-images.githubusercontent.com/38523250/62208012-47679780-b3b3-11e9-97ce-55917713ab0a.png)

**Comment by andxu** [comment]
Hi @Accessibility56 , I have tested the Intellj without installing our extensions, create a new project, down arrow button to navigate to the Maven tab on the left, the IntelliJ idea use a blue background to indicate the focus(in my theme), so this problem is regarding to the integrated Maven feature shipped with IntelliJ idea itself,  you can try it following these steps:
1. Open IntelliJ IDEA application.
2. Tab till File menu and click new-->Project.
3. In the New Project dialog box, select Maven.
4. Verify keyboard focus on “Maven” menu item.

![image](https://user-images.githubusercontent.com/17075863/64510446-ccc85b00-d314-11e9-8f15-83075ce30901.png)

**Comment by andxu** [comment]
Importance of experience: Little, user can use down arrow and up arrow key to find which control is under focus.
Quality of experience: Little, the color in high contrast theme has the indication of current focus.
Alternate solutions: No alternate technologies or options can be leveraged.

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue of IntelliJ and has been linked to AAT, please help us review it, thanks!

**Comment by Accessibility56** [comment]
Logged an External bug https://youtrack.jetbrains.com/issue/IDEA-222330

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749790
